search/search_quality/assessment_tool/main_model.cpp

```diff
@@ -224,19 +224,68 @@ void MainModel::OnShowPositionClicked()
 
 bool MainModel::HasChanges() { return m_contexts.HasChanges(); }
 
+bool MainModel::AlreadyInSamples(FeatureID const & id)
+{
+  CHECK(m_selectedSample != kInvalidIndex, ());
+  CHECK(m_selectedSample < m_contexts.Size(), ());
+
+  bool found = false;
+  ForMatchingEntries(m_contexts[m_selectedSample], id, [&](Edits & edits, size_t index)
```

Comment:
FindEntry will do the same job, moreover, what do you think FindEntry should return? Note, the number of entries here is quite small.